src/Directory.Build.props

```diff
@@ -33,4 +33,7 @@
 	<ItemGroup>
 		<RuntimeHostConfigurationOption Include="System.IO.UseNet5CompatFileStream" Value="true" />
 	</ItemGroup>
+	<PropertyGroup>
+		<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
+	</PropertyGroup>
```

Comment:
```suggestion
```

it seems to build/publish ok for me locally without this change (haven't tried on ci though)

Reply:
fixed (removed after PR was split into 2) (occurs only on .NET 7.0)